src/IBM.Watson.SpeechToText.v1/Model/SpeechRecognitionAlternative.cs

```diff
@@ -42,15 +42,15 @@ public class SpeechRecognitionAlternative
         /// `[[\"hello\",0.0,1.2],[\"world\",1.2,2.5]]`. Timestamps are returned only for the best alternative.
         /// </summary>
         [JsonProperty("timestamps", NullValueHandling = NullValueHandling.Ignore)]
-        public List<List<string>> Timestamps { get; set; }
+        public string[][] Timestamps { get; set; }
```

Comment:
I assume it's fine since CI is passing, but is this change intended/okay?

Reply:
Ah yeah either will work. I had string[][] before but I forgot I transitioned to List. Ill change. Good catch!